helm-rhythmbox.el

```diff
@@ -120,6 +120,15 @@ formatted with `helm-rhythmbox-candidate-format'."
       (dbus-call-method :session service path interface
                         "AddToQueue" (rhythmbox-song-uri song)))))
 
+(defun helm-rhythmbox-pause-song ()
```

Comment:
Maybe incorporate `play` in the title, something like `helm-rhythmbox-play/pause-song`?